src/cli/runs/kill.rs

```diff
@@ -0,0 +1,30 @@
+use clap::Parser;
+
+use crate::{runs::DaemonRunsManager, Project};
+
+/// Kill a detached run. It will send a SIGTERM signal to the process.
+#[derive(Parser, Debug)]
+pub struct Args {
+    /// The name of the run to kill
+    pub name: String,
+}
+
+pub async fn execute(project: Project, args: Args) -> miette::Result<()> {
+    // Init the runs manager
+    let runs_manager = DaemonRunsManager::new(&project);
+
+    // Get the run
+    let run = runs_manager.get_run(args.name)?;
+
+    // Kill the run
+    run.kill()?;
```

Comment:
When calling that cli on the jupyterlab example, the parent pixi process get killed but the child `jupyter lab` is still alive (I can continue using it).